pkg/infra/cli.go

```diff
@@ -128,8 +129,8 @@ func GetLiveState(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-
-	result, err := reader.ReadState(stateBytes, input.Graph)
+	bytesReader := bytes.NewReader(stateBytes)
```

Comment:
Instead of `ReadFile` earlier, I mean using `os.Open`. Sorry for the back-and-forth, I could've explained it better